config/config.go

```diff
@@ -45,9 +46,12 @@ func (c *Config) MakeAPICall(verb string, endpoint string, body []byte) (*http.R
 	var request *http.Request
 	var reqErr error
 
-	url := c.Endpoint + endpoint
+	url, err := url.JoinPath(c.Endpoint, endpoint)
```

Comment:
Might want to rename the url variable to something else since the package is also `"net/url"` to avoid confusion, but not necessary.